backend/src/routes/taskLabelRoute.js

```diff
@@ -0,0 +1,37 @@
+import { Router } from "express";
+import authorize from "../auth/authorize.js";
+import TaskLabelController from "../controllers/taskLabelController.js";
+
+const router = Router();
+const taskLabelController = new TaskLabelController();
+
+router.post("/", authorize(), (req, res) => {
```

Comment:
I'm thinking maybe it's best if we combine all of these into the labelRoute, just so we have less moving pieces and the functionality are all very related.